### DISCUSSION TITLE
PiJuice and Anavi Infrared pHAT i2c conflict

### DISCUSSION THREAD

**Discussion by Meridien** [PiSupply/PiJuice#828] [Q&A] [answered]
Hi everyone,

I want to stack ANAVI Infrared pHAT on top of PiJuice (http://anavi.technology/files/anavi-infrared-phat.pdf). 

Infrared pHAT wants to use:

```
I2C: 3, 5
EEPROM: 27, 28
IR LED: 11
IR receiver: 12
UART: 1, 8, 9, 10
```

Seems PiJuice also wants to use I2C 3 and 5 (as per https://pinout.xyz/pinout/pijuice)

So - I have gone to reconfigure the PiJuice I2Cs in pijuice_cli, except when I get there it has "i2c address: 14". If I know another pin I can assign this to (e.g. 22, 36, 37 are not in use from the Infrared pHAT) how do I translate that to an "i2c address"?

I have tried googling this but not having much luck, seems like one of those things that is assumed knowledge.

Thank you,
Daniel

**Comment by tvoverbeek** [comment, marked as answer]
I2C is a bus system. You can connect many sensors in parallel as long as there are no address  conflicts.
Pijuice uses by default addresses 0x14 and 0x68 (for the RTC).
The ANAVI hat  allows you to attach  3 I2C sensors in parallel.
The mentioned examples on the anavi site have no address conflicts with the PiJuice:
BMP180 uses 0x77
HTU21 uses 0x40
BH1750 uses 0x23
So no problem to use them stacked on each other.

**Reply by Meridien** [reply]
Thanks very much for your reply, I can confirm that it works with no issue.

What threw me was when I first stacked pijuice on the pi, with Anavi on top and plugged in the power I did not get the expected flashing LEDs (just a solid red LED1). I removed the Anavi and plugged in power again and got the expected flashing LEDs, so assumed there was some conflict in the GPIO.
